src/openvic-simulation/politics/Government.hpp

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include "openvic-simulation/types/IdentifierRegistry.hpp"
+#include "openvic-simulation/politics/Ideology.hpp"
+
+namespace OpenVic {
+    struct GovernmentTypeManager;
+
+    struct GovernmentType : HasIdentifier {
+        friend struct GovernmentTypeManager;
+
+    private:
+        std::vector<std::string> ideologies;
+        const bool elections, appoint_ruling_party;
+        const uint16_t election_duration;
+        const std::string_view flag_type_identifier;
+
+        GovernmentType(std::string_view new_identifier, std::vector<std::string> new_ideologies, bool new_elections, bool new_appoint_ruling_party, uint16_t new_election_duration, std::string_view new_flag_type_identifier);
+
+    public:
+        GovernmentType(GovernmentType&&) = default;
+
+        bool is_ideology_compatible(std::string_view ideology) const;
```

Comment:
```suggestion
        bool is_ideology_compatible(Ideology const* ideology) const;
        std::vector<Ideology const*> const& get_ideologies() const;
```
Again, avoid identifier lookups where possible (especially post-load), also add a getter for the sake of completeness.